library/locale.po

```diff
@@ -80,20 +104,27 @@ msgid ""
 "locale is not changed thereafter, using multithreading should not cause "
 "problems."
 msgstr ""
+"Cela positionne la locale dans toutes les catégories sur le paramètre par "
```

Comment:
*locale* is an english word, have you tried translating it?

Reply:
Yep, I'll go with « les paramètres régionaux », thanks :).